FantasmoSDK/app/build.gradle

```diff
@@ -49,26 +51,35 @@ android {
 
     flavorDimensions "env"
     productFlavors {
+        buildTypes{
+            release{
+                buildConfigField "String", "FM_API_BASE_URL", "\"https://api.fantasmo.io/v1/image.localize\""
+                buildConfigField "String", "FM_LOCALIZATION_AVAILABLE_URL", "\"https://mobility-bff.fantasmo.dev/v2/isLocalizationAvailable\""
+                buildConfigField "String", "FM_INITIALIZE_URL","\"https://mobility-bff.fantasmo.dev/v2/initialize\""
+            }
+            debug{
+                buildConfigField "String", "FM_API_BASE_URL", "\"https://api-dev.fantasmo.io/v1/image.localize\""
+                buildConfigField "String", "FM_LOCALIZATION_AVAILABLE_URL", "\"https://mobility-bff-dev.fantasmo.dev/v2/isLocalizationAvailable\""
+                buildConfigField "String", "FM_INITIALIZE_URL","\"https://mobility-bff-dev.fantasmo.dev/v2/initialize\""
+            }
+        }
         prod{
             dimension "env"
             buildConfigField "String", "VERSION_NAME", "\"$defaultConfig.versionName\""
         }
         devParis {
```

Comment:
also are these build flavours still needed? My understanding is that we no longer offer separate `.arr`files for these setups?

Reply:
The flavors are no longer needed. We left them in build.gradle but I will remove them